lib/Core/Executor.cpp

```diff
@@ -4370,9 +4370,7 @@ KValue Executor::handleReadForLazyInit(ExecutionState &state,
         klee_warning("MaxPointerDepth reached, stopping the fork");
         result = {0, constantZero};
       } else {
-        Expr::Width typeWidth = getWidthForLLVMType(target->inst->getType());
-        const Array* array = CreateArrayWithName(state, typeWidth, "lazy_init_arr");
-        ref<Expr> size = Expr::createTempRead(array, typeWidth);
+        ref<Expr> size = getPointerSymbolicSizeExpr(state);
```

Comment:
The name of the commit is misleading, could you rename it e.g., to `extract method for creating/getting symbolic size` or something like that?